appcues/src/main/java/com/appcues/trait/TraitKoin.kt

```diff
@@ -27,6 +28,7 @@ internal object TraitKoin : KoinScopePlugin {
         factory { params -> BackdropKeyholeTrait(params.getOrNull()) }
         factory { params -> ModalTrait(params.getOrNull(), params.get(), get()) }
         factory { params -> TooltipTrait(params.getOrNull(), params.get(), get()) }
+        factory { params -> EmbedTrait(params.getOrNull(), params.get(), get()) }
```

Comment:
based on prior Koin convo and [change in PR](https://github.com/appcues/appcues-android-sdk/pull/434/commits/de3818eda79d1439e85bf9095bae7630ae3f2bfc#diff-f0670ef1ed9e62e4df09f5d2a4cc4680206e0a3629a9ff828f1dec73260c174a), we'll want to evolve to
```
EmbedTrait(it[0], it[1], get())
```